src/hardware/sblaster.cpp

```diff
@@ -1059,938 +1189,1332 @@ static void set_channel_rate_hz(const uint32_t requested_rate_hz)
+{
+	dest[0] = ((((value)&0xf0) >> 3) | (sb.type == SBType::SB16 ? 1 : 3));
+	dest[1] = ((((value)&0x0f) << 1) | (sb.type == SBType::SB16 ? 1 : 3));
+}
 
-#define MAKEPROVOL(_WHICH_)											\
-	((((_WHICH_[0] & 0x1e) << 3) | ((_WHICH_[1] & 0x1e) >> 1)) |	\
-		((sb.type==SBT_PRO1 || sb.type==SBT_PRO2) ? 0x11:0))
+static uint8_t make_sb_pro_volume(const uint8_t* src)
+{
+	return ((((src[0] & 0x1e) << 3) | ((src[1] & 0x1e) >> 1)) |
+	        ((sb.type == SBType::SBPro1 || sb.type == SBType::SBPro2) ? 0x11 : 0));
+}
 
-static void DSP_ChangeStereo(bool stereo) {
+static void dsp_change_stereo(const bool stereo)
+{
 	if (!sb.dma.stereo && stereo) {
 		set_channel_rate_hz(sb.freq / 2);
-		sb.dma.mul*=2;
-		sb.dma.rate=(sb.freq*sb.dma.mul) >> SB_SH;
-		sb.dma.min=(sb.dma.rate*3)/1000;
+		sb.dma.mul *= 2;
+		sb.dma.rate = (sb.freq * sb.dma.mul) >> SbShift;
+		sb.dma.min  = (sb.dma.rate * 3) / 1000;
+
 	} else if (sb.dma.stereo && !stereo) {
 		set_channel_rate_hz(sb.freq);
-		sb.dma.mul/=2;
-		sb.dma.rate=(sb.freq*sb.dma.mul) >> SB_SH;
-		sb.dma.min=(sb.dma.rate*3)/1000;
+		sb.dma.mul /= 2;
+		sb.dma.rate = (sb.freq * sb.dma.mul) >> SbShift;
+		sb.dma.min  = (sb.dma.rate * 3) / 1000;
 	}
-	sb.dma.stereo=stereo;
+
+	sb.dma.stereo = stereo;
 }
 
-static void CTMIXER_Write(uint8_t val) {
+static void ctmixer_write(const uint8_t val)
+{
 	switch (sb.mixer.index) {
-	case 0x00:		/* Reset */
-		CTMIXER_Reset();
-		LOG(LOG_SB,LOG_WARN)("Mixer reset value %x",val);
-		break;
-	case 0x02:		/* Master Volume (SB2 Only) */
-		SETPROVOL(sb.mixer.master,(val&0xf)|(val<<4));
-		CTMIXER_UpdateVolumes();
-		break;
-	case 0x04:		/* DAC Volume (SBPRO) */
-		SETPROVOL(sb.mixer.dac,val);
-		CTMIXER_UpdateVolumes();
-		break;
-	case 0x06:		/* FM output selection, Somewhat obsolete with dual OPL SBpro + FM volume (SB2 Only) */
-		//volume controls both channels
-		SETPROVOL(sb.mixer.fm,(val&0xf)|(val<<4));
-		CTMIXER_UpdateVolumes();
-		if(val&0x60) LOG(LOG_SB,LOG_WARN)("Turned FM one channel off. not implemented %X",val);
-		//TODO Change FM Mode if only 1 fm channel is selected
-		break;
-	case 0x08:		/* CDA Volume (SB2 Only) */
-		SETPROVOL(sb.mixer.cda,(val&0xf)|(val<<4));
-		CTMIXER_UpdateVolumes();
-		break;
-	case 0x0a:		/* Mic Level (SBPRO) or DAC Volume (SB2): 2-bit, 3-bit on SB16 */
-		if (sb.type==SBT_2) {
-			sb.mixer.dac[0]=sb.mixer.dac[1]=((val & 0x6) << 2)|3;
-			CTMIXER_UpdateVolumes();
+	case 0x00: // Reset
+		ctmixer_reset();
+		LOG(LOG_SB, LOG_WARN)("Mixer reset value %x", val);
+		break;
+
+	case 0x02: // Master Volume (SB2 Only)
+		set_sb_pro_volume(sb.mixer.master, (val & 0xf) | (val << 4));
+		ctmixer_update_volumes();
+		break;
+
+	case 0x04: // DAC Volume (SBPRO)
+		set_sb_pro_volume(sb.mixer.dac, val);
+		ctmixer_update_volumes();
+		break;
+
+	case 0x06: // FM output selection
+	           // Somewhat obsolete with dual OPL SBpro + FM volume (SB2 Only)
+		// volume controls both channels
```

Comment:
Correct comment formatting.

Reply:
This was clang-format being a bit confused, actually 😄